starter_code/express-spotify-app/views/artists.ejs

```diff
@@ -0,0 +1,10 @@
+<h1>
+  <%= body.artists.items[0].name %>
```

Comment:
You are only showing the first result in the query, you need to use a for, forEach... If you did that to avoid the error of not being able to show url of undefined (some results doesn't have images), that doesn't solve the problem (try to search Japanese Beatles) and it's not the desired behaviour. You can solve this problem (and similar ones) with:
```
<% if(artists[i].images[0]){%>
  <img src="<%=artists[i].images[0].url%>" alt="" class="artist-img">
<%} %>
```